scripts/set_config.py

```diff
@@ -212,10 +219,7 @@ def create_config_obj(argv, log=None):
 
 if __name__ == '__main__':
 
-    cli_args = parse_args(sys.argv[1:])
-    LOG_NAME = "set_config"
-    cli_log = cli_helpers.setup_logging(cli_args, log_name=LOG_NAME)
     try:
-        create_config_obj(sys.argv[1:], cli_log)
+        create_config_file(sys.argv[1:])
```

Comment:
The log handler was already doing it's thing inside `create_config_file`, so I removed this duplicate work.